doc/Onboarding/Markup/Overview.md

```diff
@@ -0,0 +1,169 @@
+---
+uid: Onboarding.Markup.CSharpMarkup
+---
+
+# C# Markup
+
+## Overview
+
+C# Markup is a declarative, fluent-style syntax for defining the layout of an application in C#. With C# Markup you can define both the layout and the logic of your application using the same language. C# Markup leverages the same underlying object model as XAML, meaning that it has all the same capabilities such as data binding, converters, and access to resources. You can use all the built-in controls, any custom controls you create, and any 3rd party controls all from C# Markup.
+
+You will quickly discover why C# Markup is a developer favorite with:
+
+- A Declarative syntax
+- Strongly typed data binding
+- Intellisense and compile time validation
+- Refactoring support
+- Custom Controls and 3rd party libraries
+
+Let's jump in and take a look at a simple sample that displays 'Hello Uno Platform!' in the center of the screen:
+
+```cs
+public sealed partial class MainPage : Page
+{
+    public MainPage()
+    {
+        this.Content
+        (
+            new StackPanel()
+                .VerticalAlignment(VerticalAlignment.Center)
+                .HorizontalAlignment(HorizontalAlignment.Center)
+                .Children
+                (
+                    new TextBlock()
+                        .Text("Hello Uno Platform!")
+                )
+        );
+    }
+}
+```
+
+The first thing you will notice with C# Markup is that there is nothing special to learn. You can simply create a new instance of the controls you want to work with (eg `new TextBlock()`) and set properties using the generated extension method with the same name (eg `.Text("Hello Uno Platform!')`). 
+
+Unlike XAML which is littered with string constants, C# Markup provides a strongly typed API for setting properties. For example, in XAML you would set HorizontalAlignment with `HorizontalAlignment="Center"`, but in C# Markup you would use the `HorizontalAlignment` enum with `HorizontalAlignment.Center`, ensuring that you get compile time validation and intellisense for all the available values.
+
+## Getting Started
+
+Let's take a look at how to get started with C# Markup. We'll start with a simple sample that displays a counter and a button that increments the counter by a step size. We'll start with a simple ViewModel that has a `CounterValue` and a `StepSize` property, as well as a `IncrementCommand` that increments the `CounterValue` by the `StepSize` when executed. 
```

Comment:
You say that "We'll start with a simple ViewModel" and the ViewModel is never shown, should it be here?

```cs
internal partial class MainViewModel : ObservableObject
{
    [ObservableProperty]
    private int _count = 0;

    [ObservableProperty]
    private int _step = 1;

    [RelayCommand]
    private void Increment()
        => Count += Step;
}
```

Reply:
We don't want to get into the actual viewmodel here. I've updated the text slightly to make it clearer (hopefully)